flags/client.ts

```diff
@@ -0,0 +1,11 @@
+import {
+	createUseFlags,
```

Comment:
Can you run prettier on these? Looks like some files have tabs and others have spaces.

Reply:
https://github.com/hashicorp/terraform-website/pull/2421/commits/dbb6a8a872beac206f261842551bf06e4917176a